elixir/apps/domain/lib/domain/policies/policy/changeset.ex

```diff
@@ -14,8 +14,8 @@ defmodule Domain.Policies.Policy.Changeset do
     |> cast_embed(:conditions, with: &Domain.Policies.Condition.Changeset.changeset/3)
     |> changeset()
     |> put_change(:account_id, subject.account.id)
-    |> put_change(:created_by, :identity)
-    |> put_change(:created_by_identity_id, subject.identity.id)
+    |> put_change(:created_by_actor_id, subject.actor.id)
```

Comment:
Let's move this to `put_created_by` too.

Reply:
👍 Sounds good